test/helper/test-function.ts

```diff
@@ -41,6 +41,13 @@ export function testFilterFunction(
 	describe(`Test '${functionName}' filter function`, () => {
 		for (const data of testData) {
 			const { description, funcParameter, expectedValue } = data;
+			if (
```

Comment:
I feel this check should be done in the `loadFixtureFile` function, although it requires 2x loops. Does this change increase executing time significantly?

Reply:
Well I see `loadFixtureFile` is a generic, so we could make a wrapper which returns the FunctionTestData implementations directly.

Reply:
I think we'd need to be dealing with thousands of test cases before we start noticing a performance impact from processing each test case twice. But yeah, the reason I didn't do it in `loadFixtureFile` was because it was meant to be generic in case other fixture files are added in future. I'd be happy to add a wrapper function that validates the data before passing it to `testFilterFunction`.

And yeah, I was wondering if you'd want to combine everything back into a single file but I forgot to bring it up. I'll make that change too.